docs/grammar/SolidityLexer.g4

```diff
@@ -315,6 +321,7 @@ YulDecimalNumber: '0' | ([1-9] [0-9]*);
 YulStringLiteral:
 	'"' DoubleQuotedStringCharacter* '"'
 	| '\'' SingleQuotedStringCharacter* '\'';
+//@doc:inline
```

Comment:
This fixes https://github.com/ethereum/solidity/pull/11173#discussion_r609708515